### PR TITLE
Remove duplicate getParquetType method from ParquetPageSourceFactory

### PR DESCRIPTION
**Pull request by leetcode-1533**
## Description

Remove duplicate getParquetType method from ParquetPageSourceFactory

## Release notes

(x) This is not user-visible or docs only and no release notes are required.
( ) Release notes are required, please propose a release note for me.
( ) Release notes are required, with the following suggested text:

```markdown
# Section
* Fix some things. ({issue}`issuenumber`)
```

### CODE REVIEW

File: plugin/trino-hive/src/main/java/io/trino/plugin/hive/parquet/ParquetPageSourceFactory.java [hunk 1]
```diff
@@ -56,6 +56,7 @@
 import org.apache.parquet.io.MessageColumnIO;
 import org.apache.parquet.schema.GroupType;
 import org.apache.parquet.schema.MessageType;
+import org.apache.parquet.schema.Type;
```
Comment (raunaqmorarka) [inline plugin/trino-hive/src/main/java/io/trino/plugin/hive/parquet/ParquetPageSourceFactory.java:59]: Please keep the full package name in use. Without that it is easy to confuse parquet type with trino type

File: plugin/trino-hive/src/main/java/io/trino/plugin/hive/parquet/ParquetPageSourceFactory.java [hunk 2]
```diff
@@ -521,4 +498,16 @@ public static ConnectorPageSource createParquetPageSource(
 
         return pageSourceBuilder.build(parquetReaderProvider.createParquetReader(parquetColumnFieldsBuilder.build()));
     }
+
+    private static Optional<Type> getBaseColumnType(HiveColumnHandle column, MessageType messageType, boolean useParquetColumnNames)
```
Comment (raunaqmorarka) [inline plugin/trino-hive/src/main/java/io/trino/plugin/hive/parquet/ParquetPageSourceFactory.java:502]: getBaseColumnType -> getBaseColumnParquetType

File: plugin/trino-hive/src/main/java/io/trino/plugin/hive/parquet/ParquetPageSourceFactory.java (hunk 1 above)
Comment (leetcode-1533) [reply plugin/trino-hive/src/main/java/io/trino/plugin/hive/parquet/ParquetPageSourceFactory.java:59]: sounds good

File: plugin/trino-hive/src/main/java/io/trino/plugin/hive/parquet/ParquetPageSourceFactory.java (hunk 2 above)
Comment (leetcode-1533) [reply plugin/trino-hive/src/main/java/io/trino/plugin/hive/parquet/ParquetPageSourceFactory.java]: done.

**Comment by leetcode-1533** [comment]
hi, updated, let me know if there is more issues!

**Review by raunaqmorarka — APPROVED** [review]

**Pull request merged**